Dockerfile.test.db

```diff
@@ -15,6 +15,8 @@ COPY raisinbread/instruments/instrument_sql/mri_parameter_form.sql  /mri_paramet
 COPY raisinbread/instruments/instrument_sql/radiology_review.sql  /radiology_review.sql
 COPY test/test_instrument/testtest.sql /test_instrument.sql
 COPY raisinbread/RB_files/*.sql /RB_files/
+COPY raisinbread/instruments/*.inc /project/instruments/
+COPY raisinbread/instruments/*.linst /project/instruments/
```

Comment:
There lines will copy the files into loris_db container. It will not work for loris.
```suggestion
```